cypress/e2e/lotto.cy.js

```diff
@@ -158,6 +167,98 @@ describe("행운의 로또 테스트", () => {
     });
   });
 
+  describe("소비자는 수동 구매를 할 수 있어야 한다.", () => {
+    it("수동 입력 추가하기 버튼이 존재한다.", () => {
+      cy.get($createManualLottoButtonSelector).should("exist");
+    });
+
+    describe("수동 입력 추가하기 버튼을 클릭하면 수동으로 입력할 수 있는 게임이 생성된다", () => {
+      beforeEach(() => {
+        cy.get($createManualLottoButtonSelector).click();
+      });
+
+      it("게임당 6개의 번호를 수동으로 입력할 수 있는 Input이 나타난다.", () => {
+        cy.get($manualNumberInputSelector).should("have.length", 6);
+      });
+
+      it("수동 입력 추가하기 버튼을 클릭할때마다 수동 게임이 추가된다.", () => {
```

Comment:
(마이너) 요런식으 네이밍은 어떨까요 👀 
```suggestion
      it("수동 입력 추가하기 버튼을 클릭하면 수동으로 로또를 입력할 수 있는 Input이 추가로 렌더링된다", () => {
```

Reply:
@EungyuCho 크큭.. 사실 수동쪽 테스트 케이스 작성할때 어떻게 네이밍을 지어야될지 헷갈렸습니다.. 어떻게 작성해도 뭔가 이상하게 이질감이 들더라구요.. 좋은 것 같습니다 '수동 게임이 추가된다'이게 명확하지가 않네요 데이터가 추가되는건지 UI가 추가되는건지
내부 테스트 로직은 UI를 체크하는데 말이죠.. 의견 감사합니다!

리뷰 반영 커밋
https://github.com/next-step/js-lotto/pull/208/commits/41c9dc5f11722ddd8b2d1079bac6bbdd1c70c7f7